src/components/EventDetails.js

```diff
@@ -1,39 +1,43 @@
 import React from 'react';
 import { Button } from './Button';
 import { SimpleMap } from './SimpleMap';
+import { Spinner } from './Spinner';
 
 export const EventDetails = (props) => {
-  return (
-    <div className='event-details-card'>
-      <div className="event-details-container">
-        <h3>{props.event.name.text}</h3>
-        {
-          props.event.logo.url ? 
-          <img alt="Event" src={props.event.logo.url} />
-          : null
-        }
+  return(
+    props.eventError ?  <div>Sorry, there seems to be an error. Please try again later.</div>
```

Comment:
I agree with @cgolinski, we can have a renderContent function. Having ternary inside ternary is hard to read. Or at least we should create 2 functions to render content. 
renderError = () => {return <div>Sorry, there seems to be an error. Please try again later.</div>} and renderContainer = () => {return props.eventFetching ? <div className="overlay"><Spinner/></div>  : <div className='event-details-card'> }